lib/weightio.c

```diff
@@ -121,7 +121,7 @@
 MTC_STATUS
 open_hostweight_file(int *fd, int *err_no)
 {
-    if ((*fd = open(HA_HOST_WEIGHT_FILE, O_RDWR|O_CREAT)) < 0)
+    if ((*fd = open(HA_HOST_WEIGHT_FILE, O_RDWR|O_CREAT, S_IRUSR | S_IWUSR)) < 0)
```

Comment:
Can we do this in octal so people can sensibly read the perms?

Reply:
IRUSR is user read and IWUSR is user write, so this would be `0600`, but thought it is more readable this way (it is waht the manpage recommends)

Reply:
Changed to octal.